Add tutorial: Clade Workload Identty integration with Agent ID (#6132)
wso2/docs-is: en/includes/tutorials/agent-id-claude-authentication.md

> # Authenticate Agents with their own Agent ID to Claude API Using Workload Identity Federation

Reviewer: ```suggestion
# Authenticate AI Agents with WSO2 Agent ID to Claude API Using Workload Identity Federation
```

> This tutorial walks you through connecting an AI agent registered in {{product_name}} to the Claude API using [Anthropic's Workload Identity Federation (WIF)](https://docs.anthropic.com/en/docs/build-with-claude/workload-identity-federation/). By the end of this tutorial, your agent will authenticate with {{product_name}}, obtain a signed JWT, exchange it for a short-lived Claude access token, and make API calls without any static API keys.

Reviewer 2: ```suggestion
This tutorial walks you through connecting an AI agent registered in {{product_name}} to the Claude API using [Anthropic's Workload Identity Federation (WIF)](https://docs.anthropic.com/en/docs/build-with-claude/workload-identity-federation/). By the end of this tutorial, your agent will authenticate with {{product_name}}, receive a signed JWT, exchange it for a short-lived Claude access token, and make API calls without any static API keys.
```

## Overview

The authentication flow works as follows:

1. Your AI agent authenticates with {{product_name}} using its Agent ID and Secret.

> 2. {{product_name}} issues a signed JWT (access token) to the agent.

Reviewer: ```suggestion
2. {{product_name}} issues a signed JWT access token to the AI agent.
```

> 3. The agent presents this JWT to Anthropic's token endpoint.

Reviewer: ```suggestion
3. The AI agent presents this JWT token to Anthropic's token endpoint.
```

4. Anthropic validates the JWT and returns a short-lived access token.
5. The agent uses the access token to call the Claude API.

## Prerequisites

Before you begin, ensure you have the following:

- An [Anthropic](https://console.anthropic.com/) account with access to the Claude Console.
- `curl` and `jq` installed on your machine (for the shell-based approach).
- Python 3.9+ (for the SDK-based approach).

> ## Step 1: Register an Agent in {{product_name}}

Reviewer: ```suggestion
## Step 1: Register your AI Agent in {{product_name}}
```

> ## Step 1: Register your AI Agent in {{product_name}}

Reviewer 2: Each step can benefit from having a small description as to why we are doing each step. Shall we do that?

1. Log in to the {{product_name}} Console.

> 2. Navigate to **Agentic AI**.

Reviewer: ```suggestion
2. Navigate to **Agents**.
```

> 3. Click **New Agent** and provide a descriptive name for your agent.

Reviewer: ```suggestion
3. Click **New Agent** and provide a descriptive name for your AI agent.
```

> 4. Assign the agent to appropriate groups and roles based on the resources it needs to access.

Reviewer: This is not possible during agent creation, is it?

4. Note down the **Service Account ID** (format: `svac_...`).

## Step 5: Create a Federation Rule

1. In the Claude Console, go to **Settings > Workload Identity > Federation Rules**.
2. Click **Create Rule** and configure:
   - **Issuer**: Select the `{{product_name}}-agents` issuer you created in Step 3.
   - **Subject claim**: Set this to the Agent ID from {{product_name}} (the `sub` claim value in the JWT).
   - **Audience**: `https://api.anthropic.com`
   - **Service Account**: Select the service account you created in Step 4.
   - **Token lifetime**: `600` seconds (10 minutes).
3. Enable the rule for the workspace(s) where the agent needs access.
4. Click **Save** and note down the **Federation Rule ID** (format: `fdrl_...`).

!!! note
    If you enable the rule across multiple workspaces, you must include the `workspace_id` parameter in the token exchange request. Without it, Anthropic returns an error: `"rule is enabled for more than one workspace; pass workspace_id to choose one"`.

## Step 6: Authenticate Your Agent

With all the configuration in place, your agent can now authenticate and call the Claude API.

> 

Reviewer: It'd be better to have a transition sentence or paragraph to smoothly link the steps and the try-out approaches.

Author: addressed
